
Basic Local Alignment Search Tool,  Journal of Molecular Biology
Vol. 215, No. 3, October 5, 1990, pp. 403-410 (doi:10.1006/jmbi.1990.9999) 

Stephen F. Altschul1,  Warren Gish1,  Webb Miller2,  Eugene W. Meyers3,  David J. Lipman1 
A new approach to rapid sequence comparison, basic local alignment search tool (BLAST), directly approximates alignments that optimize a measure of local similarity, the maximal segment pair (MSP) score. Recent mathematical results on the stochastic properties of MSP scores allow analysis of the performance of this method as well as the statistical significance of alignments it generates. The basic algorithm is simple and robust; it can be implemented in a number of ways and applied in a variety of contexts including straight-forward DNA and protein sequence database searches, motif searches, gene identification searches, and in the analysis of multiple regions of similarity in long DNA sequences. In addition to its flexibility and tractability to mathematical analysis, BLAST is an order of magnitude faster than existing sequence comparison tools of comparable sensitivity. 
